assembl/static2/js/app/components/synthesis/actions.jsx

```diff
@@ -13,11 +13,11 @@ import EditPostButton from '../debate/common/editPostButton';
 import { getDiscussionSlug } from '../../utils/globalFunctions';
 import { DELETE_CALLBACK } from '../../constants';
 
-function userCanEdit(): boolean {
+export function userCanEdit(): boolean {
```

Comment:
you export those two functions, where are they used since I don't see any use case in the PR ?

Reply:
to be fixed

Reply:
fixed